backend/config/curation-api.yml

```diff
@@ -61,12 +61,12 @@ paths:
 
         When the visibility parameter is set to PRIVATE, a list of all private Collections or private revisions of 
         public Collections that the user is authorized to access is returned. An access token MUST be set in the request
-        header as: `Authentication: bearer <access_token>`
+        header as: `Authentication: Bearer <access_token>`
```

Comment:
@Bento007 instances I'm seeing online all use `Bearer`... not that it really matters...? It's what I've been using in my testing.

Reply:
although I recognize that we're just splicing all of that out anyways... unsure if it could have functional implications at some point though

Reply:
I don't think it will have any functional implications in our stack, but using `Bearer` could save us from weird bugs in the future if we change our stack.